fbbt/src/ontology/fbbt-edit.obo

```diff
@@ -54156,7 +54156,9 @@ id: FBbt:00005783
 name: midgut interstitial cell
 def: "Interstitial cell of the midgut. There are 30-40 cells that initially delaminate from the tip of the posterior midgut rudiment during embryonic stages 10 and 11. Later on, they intercalate into the midgut epithelium." [FlyBase:FBrf0041814]
 subset: cur
+synonym: "absorptive cell of the middle midgut" NARROW [FlyBase:FBrf0208023] {comment="The authors refer to interstitial cells as \"absorptive cells\" to distinguish them from copper cells, although \"absorptive cells\" is also a synonym for enterocytes in general."}
```

Comment:
For compatibility with OBO format, please move this text to an annotation axiom directly attached to the term, using the rdfs:comment annotation property.